.scripts/create_upgrade.php

```diff
@@ -75,14 +66,44 @@
 
 $upgrade_code = <<<___UPGRADE
 <?php
```

Comment:
Yeah, that way would also be easy to have two templates; one for the simple upgrade and one for batch upgrade.